src/multi-select-item/multi-select-item.component.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes } from 'react';
+import { Checkbox } from 'react-bootstrap';
+import './multi-select-item.component.scss';
+
+export default class MultiSelectItem extends React.PureComponent {
```

Comment:
Do not extend PureComponent and implement shouldComponentUpdate at the same time